packages/components/tabs/src/tab-nav.tsx

```diff
@@ -82,6 +82,8 @@ const TabNav = defineComponent({
     const nav$ = ref<HTMLDivElement>()
     const el$ = ref<HTMLDivElement>()
 
+    const tabBar = ref<InstanceType<typeof TabBar>>()
```

Comment:
The variable name may be changed to `tabBarRef`, which looks easy to be confused with the component name `TabBar`.

Reply:
ok